.github/workflows/maya-usd-preflight-launcher.yml

```diff
@@ -9,28 +9,51 @@ on:
 
 jobs:
 
-  build:
-      timeout-minutes: 180
+  # Wait for remote build to start and finish, report results
+  build_preflight:
+      timeout-minutes: 400
       runs-on: ubuntu-latest
       if: contains(toJson(github.event.pull_request.assignee), 'ecp-maya-devops-adsk')
       steps:
 
-        # Build logs and results will be committed here when the remote build is complete
+        # Build start info will be committed here when the remote build is launched
         - name: Setup transfer repo
           uses: actions/checkout@v2
           with:
             repository: ecp-maya-devops-adsk/log-transfer
             ref: transfer
             path: transfer
 
+        # Echo the file name - it's hard to find the run_id in the UI
+        - name: Echo the expected start file name
+          run: "echo ${{ github.run_id }}_${{ github.run_number }}_start.txt"
+
+        # Wait for remote build to start
+        - name: Wait until remote build starts
+          shell: bash
+          # 1080 = 180 minutes in 10 second increments. There will be overhead for the git pull command, so wait time will be more than 180 minutes
```

Comment:
From the end-user perspective, I don't really care if this sleep will be 10 seconds or 1 minute. How about we reduce the frequency of these calls?

BTW. Typically we count the time that is elapsed, not the number of iterations. Is this something hard to achieve here?

Reply:
That's a good idea. 1-minute sleeps is vastly more intuitive. I'll add that now.

We could be more precise with the timing, but I think the rough count is sufficient for the timeout purpose.